pynest/examples/gap_junctions_inhibitory_network.py

```diff
@@ -22,29 +22,27 @@
 """
 Gap Junctions: Inhibitory network example
 ------------------
-This is the inhibitory network used as test case 2 (see figure 9 and 10) in
-
-    Hahne, J., Helias, M., Kunkel, S., Igarashi, J.,
-    Bolten, M., Frommer, A. and Diesmann, M.,
-    A unified framework for spiking and gap-junction interactions
-    in distributed neuronal network simulations,
-    Front. Neuroinform. 9:22. (2015),
-    doi: 10.3389/fninf.2015.00022
-
-The network contains 500 hh_psc_alpha_gap neurons with random initial
-membrane potentials between -40 and -80 mV. Each neuron receives 50
-inhibitory synaptic inputs that are randomly selected from all other
-neurons, each with synaptic weight JI = -50.0 pA and synaptic delay
-d = 1.0 ms. Each neuron receives an excitatory external Poissonian
-input of 500.0 Hz with synaptic weight JE = 300.0 pA and the same
-delay d. In addition (60*500)/2 gap junctions are added randomly to the
-network resulting in an average of 60 gap-junction connections per neuron.
+This script simulates an inhibitory network of `500` Hodgkin-Huxley neurons.
```

Comment:
I don't think that setting 500 in backticks will improve readability. I would use backticks only for actual code examples. This applies to the remainder of the text.